static/skin/index.js

```diff
@@ -407,6 +424,12 @@
         document.querySelectorAll('.filter').forEach(filter => {
             filter.addEventListener('change', () => {resetAndFilter(filter.name, filter.value)});
         });
+        document.querySelectorAll('.tagFilterLabel').forEach(tagElement => {
+            tagElement.addEventListener('click', (e) => {
+                tagElement.style.display = 'none';
+                resetAndFilter('tag', '');
+            });
+        });
```

Comment:
This code is a little confusing. It creates a currently wrong impression that there can be multiple tag elements.
It would be better to have a function `removeTagFilter()` right below `addTagElement()`. Then, instead of dynamically binding an event listener to the (only) tag label, you can specify the handler through the `onclick` attribute in HTML.